dev-libs/libsigc++/libsigc++-2.10.0.ebuild

```diff
@@ -16,10 +16,14 @@ IUSE="doc static-libs test"
 RDEPEND=""
 DEPEND="sys-devel/m4
 	doc? ( app-doc/doxygen )
-	test? ( dev-libs/boost )"
+	test? ( dev-libs/boost[${MULTILIB_USEDEP}] )"
 # Needs mm-common for eautoreconf
 
 src_prepare() {
+	# Properly interpret --{enable,disable}-benchmark configure option (bug 599764)
+	eapply "${FILESDIR}/${P}-allow-no-benchmark.patch"
+	eautoreconf
+
```

Comment:
you have to put the `eautoreconf` after the `sed`'s but before `gnome2_src_prepare`, as they affect the `Makefile.am`'s